src/Open3D/Visualization/Rendering/Renderer.cpp

```diff
@@ -24,26 +24,25 @@
 // IN THE SOFTWARE.
 // ----------------------------------------------------------------------------
 
-#include "Renderer.h"
-
-#include "RenderToBuffer.h"
+#include "Open3D/Visualization/Rendering/Renderer.h"
 
 #include "Open3D/Geometry/Image.h"
 #include "Open3D/Utility/Console.h"
+#include "Open3D/Visualization/Rendering/RenderToBuffer.h"
 
 namespace open3d {
 namespace visualization {
 
-static ResourceLoadRequest::ErrorCallback sDefaultErrorHandler =
+static ResourceLoadRequest::ErrorCallback kDefaultErrorHandler =
```

Comment:
if we are going to prefix with "k", lets make this "static const"?

Reply:
I wasn't 100% sure how to handle this when reviewing the code. However, I interpreted the Google style guide to suggest that static, file scope variables should have the 'k' prefix: "All such variables with static storage duration (i.e. statics and globals, see Storage Duration for details) should be named this way. " I have no objection to changing to static const assuming DefaultErrorHandler is never changed. I'll have to double check that.

Reply:
changed to static const